lib/Extension/CodeTransformExtra/Rpc/OverrideMethodHandler.php

```diff
@@ -72,16 +72,19 @@ public function handle(array $arguments)
             return $this->createInputCallback($arguments);
         }
 
-        $transformedCode = $this->overrideMethod->overrideMethod(
-            SourceCode::fromString($arguments[self::PARAM_SOURCE]),
-            (string) $class->name(),
-            $arguments[self::PARAM_METHOD_NAME]
-        );
+        $newCode = $arguments[self::PARAM_SOURCE];
+        foreach ($this->methods($arguments) as $methodName) {
```

Comment:
Would prefer to to only pass the `$arguments[self::PARAM_METHOD_NAME]` to make it more clear -- also this could be simplified to simply `foreach ((array) $arguments[self::PARAM_METHOD_NAME]`

Reply:
Never type cast into an array, good to know!